src/main/java/frc/robot/subsystems/ArmSecondJoint.java

```diff
@@ -0,0 +1,164 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.SparkMaxPIDController;
+import com.spikes2212.command.genericsubsystem.smartmotorcontrollersubsystem.SparkMaxGenericSubsystem;
+import com.spikes2212.control.FeedForwardSettings;
+import com.spikes2212.control.PIDSettings;
+import com.spikes2212.control.TrapezoidProfileSettings;
+import com.spikes2212.dashboard.Namespace;
+import com.spikes2212.util.UnifiedControlMode;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import frc.robot.RobotMap;
+
+import java.util.function.Supplier;
+
+public class ArmSecondJoint extends SparkMaxGenericSubsystem {
+
+    public static final double DEGREES_PER_ROTATION = 360;
+    public static final double GEAR_RATIO = (1 / 25.0) * (12 / 28.0) * (12 / 28.0);
+
+    public static final int SECONDS_IN_MINUTE = 60;
+
+    private static ArmSecondJoint instance;
+
+    private final DutyCycleEncoder absoluteEncoder;
+    private final RelativeEncoder sparkMaxEncoder;
+
+    //voltage proportion (from 0 to 1) to be applied to the first joint when controlling the joint manually
```

Comment:
![second-futurama](https://github.com/Spikes-2212-Programming-Guild/ChargedUp2023/assets/71877371/87436340-b6d9-470d-bd35-2392ea7e4133)

(change first to second)